rpcs3/Loader/PSF.cpp

```diff
@@ -103,51 +120,51 @@ namespace psf
 		fmt::throw_exception("Invalid format (0x%x)", m_type);
 	}
 
-	registry load_object(const fs::file& stream)
+	std::pair<registry, error> load_object(const fs::file& stream)
```

Comment:
can't you make this some struct?
removes the need for the uninformative ".first" and ".second".

Reply:
Fixed.